pkgs/development/arduino/arduino-core/default.nix

```
@@ -29,7 +30,8 @@
 , udev
 }:
 
-assert withGui -> gtk2 != null;
+assert withGui -> gtk3 != null;
+assert withGui -> wrapGAppsHook != null;
```

```suggestion
assert withGui -> gtk3 != null && wrapGAppsHook != null;
```
A one-liner is cleaner & easier to understand